pyworkflow/plugin.py

```diff
@@ -583,6 +583,11 @@ def getTemplates(cls):
 
         return tempList
 
+    @classmethod
+    def getUrl(cls, prot=None):
+        """ Url for the plugin to point users to it"""
+        return cls._url
+
```

Comment:
input arg prot isn't used

Reply:
not in this base implementation but plugins could return a different url base on the protocol passed.

Reply:
@pconesa I would do this the other way around...create getUrl method in base Protocol class that by default, return the getUrl from the plugin...and then it is still open to be reimplemented by the protocol if needed.

Reply:
I thought about that but it might be tedious if you have custom url per protocol and no base class for them. That his covers xmipp need to have a tweeter url per protocol without adding a method per protocol for the url.

I'd also say that it would be very common to have some kind of pattern like:

<plugin url>/protocol
or 
<plugin url>#Protocol

that will simplify the management fo those url

What can accommodate both ideas, may be this is what you are saying, is to have a protocol.getUrl() that by default calls Plugin.getUrl(prot)...

Is that what you are saying?

Reply:
I got a bit lost with Xmipp need or the first part...but the last part is what I was suggesting.
protocol.getUrl() by default calls Plugin.getUrl(prot)

Reply:
ok, will do that, thanks @delarosatrevin

Reply:
I've made the change, somehow some commits from the other PR made it here. Sorry for that.